editor/js/Menubar.File.js

```diff
@@ -393,14 +393,14 @@ Menubar.File = function ( editor ) {
 	//
 
 	var link = document.createElement( 'a' );
-	link.style.display = 'none';
-	document.body.appendChild( link ); // Firefox workaround, see #6594
+	link.style.display = 'none'; 
```

Comment:
Is this line still necessary? I mean the link is not appended to the DOM anymore, right?